lib/command/init.js

```diff
@@ -80,7 +87,19 @@ module.exports = function (initPath) {
     return;
   }
 
+  const typeScriptconfigFile = path.join(testsPath, 'codecept.conf.ts');
+  if (fileExists(typeScriptconfigFile)) {
+    error(`Config is already created at ${typeScriptconfigFile}`);
+    return;
+  }
+
   inquirer.prompt([
+    {
+      name: 'typescript',
+      type: 'list',
```

Comment:
![image](https://user-images.githubusercontent.com/220264/183297493-d68a0e76-0cc9-4287-ac8a-580db4ec4664.png)

Please try to use `confirm` option here